### PR TITLE
Generate csv

### PR DESCRIPTION
**Pull request by dustinblack**
## Changes introduced with this PR

Adding a "power user" option to the plugin to generate the data in CSV format, output to the debug logs. This aids the plugin user in quick data ingest to spreadsheets.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: arcaflow_plugin_pcp/pcp_schema.py [hunk 1]
```diff
@@ -13,8 +13,7 @@ class PcpInputParams:
             " space-separated string."
         ),
     ] = (
-        "kernel.cpu.util kernel.all.load"
-        " mem.util.used disk.all.read disk.all.write"
+        "kernel.cpu.util kernel.all.load" " mem.util.used disk.all.read disk.all.write"
```
Comment (webbnh) [inline arcaflow_plugin_pcp/pcp_schema.py]: You should probably remove the embedded `" "` since the string is now all on one line.

**Review by webbnh — APPROVED** [review]
Looks good as far as I can tell, although I did find one nit.

File: arcaflow_plugin_pcp/pcp_plugin.py [hunk 2]
```diff
@@ -123,23 +123,38 @@ def start_pcp(
...
 
+        # Initialize (optional) pcp2csv command
+        pcp2csv_cmd = [
+            "/usr/bin/pcp2csv",
+        ]
+
+        pcp2csv_cmd.extend(pcp2_flags)
+        pcp2csv_cmd.extend(metrics)
+
+        print(f"pcp2json command: {pcp2json_cmd}")
```
Comment (dbutenhof) [inline arcaflow_plugin_pcp/pcp_plugin.py]: Seems there's no reason to do this outside the `params.generate_csv` block?

**Review by dbutenhof — APPROVED** [review]

**Review by jaredoconnell — APPROVED** [review]
Printing it doesn't seem ideal, but it works. Seems good once the other conversations are addressed.

**Pull request merged**